TypeCobol.LanguageServer.Test/ProcessorTests/DataLayoutProcessorCSVTest.cs

```diff
@@ -17,14 +19,25 @@ public class DataLayoutProcessorCSVTest
         private const string ROOT_PATH = "ProcessorTests";
         private readonly DataLayoutProcessor _processor = new();
 
-        private static void DoTestProcessor(string sourceFileName, string expectedFileName, Func<CompilationUnit, (string Header, string[] Rows)> executeProcessor, [CallerMemberName] string testName = null)
+        private static void DoTestProcessor(string sourceFileName, string testDataFileName, Func<CompilationUnit, Position, (string Header, string[] Rows)> executeProcessor, bool isCopy = false, [CallerMemberName] string testName = null)
         {
             // Parse source file
             var folder = PlatformUtils.GetPathForProjectFile(RELATIVE_PATH, Path.GetFullPath(ROOT_PATH));
-            var compilationUnit = ParserUtils.ParseCobolFile(sourceFileName, folder, execToStep: ExecutionStep.SemanticCrossCheck);
+            var compilationUnit = ParserUtils.ParseCobolFile(sourceFileName, folder, isCopy, execToStep: ExecutionStep.SemanticCrossCheck);
+
+            // Parse data file
+            var testDataFilePath = $"{Path.Combine(folder, testDataFileName)}.txt";
+            var testData = ParseContent(testDataFilePath);
+
+            // Retrieve arguments
+            var arguments = JArray.Parse(testData.Arguments).Cast<object>().ToArray(); // Array of JToken, explicit cast to object is to avoid warning on conversion when calling the constructor
+            if (arguments == null || arguments.Length == 0 || arguments[0] is not JObject jObject)
+            {
+                throw new ArgumentException($"Invalid arguments for test {testDataFileName}");
+            }
```

Comment:
Why use an array ? The `GetDataLayout` request has a single argument which is an object. If you intend to add the format, you could switch to `GetDataLayoutParams` class to represent the args

Reply:
No format in the arguments
Array can be removed